Source/Scene/PointCloud3DTileContent.js

```diff
@@ -121,7 +119,13 @@ define([
         this._translucentRenderState = undefined;
 
         this._highlightColor = Color.clone(Color.WHITE);
+
+        this.pointAttenuation = true;
+        this._pointAttenuation = true;
+
         this._pointSize = 1.0;
+        this._pointAttenuationMaxSize = 10.0;
```

Comment:
I wasn't sure if I wanted to expose much of the internals yet since the attenuation is still kind of basic and not on by default. I could see those becoming options in the future though.